azure-spring-apps-enterprise/workshops/azure-spring-apps-enterprise/intermediate/01-workshop-environment-setup/README.md

```diff
@@ -225,50 +225,45 @@ export MSYS_NO_PATHCONV=1
 
 Configure diagnostics settings
 ```shell
-az monitor diagnostic-settings create --name "send-logs-and-metrics-to-log-analytics" \ 
-    --resource ${SPRING_APPS_RESOURCE_ID} \ 
-    --workspace ${LOG_ANALYTICS_RESOURCE_ID} \ 
-    --logs '[ 
-         { 
-           "category": "ApplicationConsole", 
-           "enabled": true, 
-           "retentionPolicy": { 
-             "enabled": false, 
-             "days": 0 
-           } 
-         }, 
-         { 
-            "category": "SystemLogs", 
-            "enabled": true, 
-            "retentionPolicy": { 
-             "enabled": false, 
-             "days": 0 
-
-            } 
-
-          }, 
-
-         { 
-            "category": "IngressLogs", 
-            "enabled": true, 
-            "retentionPolicy": { 
-              "enabled": false, 
-              "days": 0 
-             } 
-           } 
-       ]' \ 
-       --metrics '[ 
-         { 
-           "category": "AllMetrics", 
-           "enabled": true, 
-           "retentionPolicy": { 
-             "enabled": false, 
-             "days": 0 
-           } 
-
-         } 
-
-       ]' 
+az monitor diagnostic-settings create --name "send-logs-and-metrics-to-log-analytics" \
+    --resource ${SPRING_APPS_RESOURCE_ID} \
+    --workspace ${LOG_ANALYTICS_RESOURCE_ID} \
+    --logs '[ 
+         { 
+           "category": "ApplicationConsole", 
+           "enabled": true, 
+           "retentionPolicy": { 
+             "enabled": false, 
+             "days": 0 
+           } 
+         }, 
+         { 
+            "category": "SystemLogs", 
+            "enabled": true, 
+            "retentionPolicy": { 
+             "enabled": false, 
+             "days": 0 
+            } 
+          }, 
```

Comment:
nit: we can format this json structure to beautify the indents and alignments.

Reply:
Formatted.